static/js/src/blog-feed.js

```diff
@@ -0,0 +1,22 @@
+function renderFeed() {
+  const container = document.querySelector('#blog-feed');
```

Comment:
Could we write this in a way that is more agnostic to the mark up? For example pass the id through the to an init function?

Reply:
Done.